0f7861669b9df34f4826f1a6085872d1a3000b5f.patch

```diff
@@ -0,0 +1,100 @@
+From 0f7861669b9df34f4826f1a6085872d1a3000b5f Mon Sep 17 00:00:00 2001
+From: suve <veg@svgames.pl>
+Date: Fri, 23 Dec 2022 14:02:06 +0100
+Subject: [PATCH] Update AppStream metadata
+
+---
+ Makefile.in                                   |  4 +-
+ ...orful.desktop => pl.suve.colorful.desktop} |  0
+ ...data.xml => pl.suve.colorful.metainfo.xml} | 37 ++++++++++++++++---
+ 3 files changed, 33 insertions(+), 8 deletions(-)
+ rename pkg/{colorful.desktop => pl.suve.colorful.desktop} (100%)
+ rename pkg/{colorful.appdata.xml => pl.suve.colorful.metainfo.xml} (55%)
+
+diff --git a/Makefile.in b/Makefile.in
+index f8d8895..91cfe9f 100644
+--- a/Makefile.in
++++ b/Makefile.in
+@@ -48,8 +48,8 @@ install: all
+ 	install -m 644 -p -D "pkg/colorful-64x64.png"   "$(DESTDIR)/usr/share/icons/hicolor/64x64/apps/colorful.png"
+ 	install -m 644 -p -D "pkg/colorful-128x128.png" "$(DESTDIR)/usr/share/icons/hicolor/128x128/apps/colorful.png"
+ 	# Desktop file and AppStream metadata
+-	install -m 644 -p -D "pkg/colorful.desktop"     "$(DESTDIR)/usr/share/applications/colorful.desktop"
+-	install -m 644 -p -D "pkg/colorful.appdata.xml" "$(DESTDIR)/usr/share/appdata/colorful.appdata.xml"
++	install -m 644 -p -D "pkg/pl.suve.colorful.desktop"      "$(DESTDIR)/usr/share/applications/pl.suve.colorful.desktop"
++	install -m 644 -p -D "pkg/pl.suve.colorful.metainfo.xml" "$(DESTDIR)/usr/share/metainfo/pl.suve.colorful.metainfo.xml"
```

Comment:
this is the wrong path too. `/usr` should be replaced by `$(PREFIX)`

for the name change you can put `rename-desktop-file` and `rename-appdata-file` in the toplevel of the manifest.